### PR TITLE
feat: `deno task init:stripe` and further documentation improvements

### PR DESCRIPTION
**Pull request by iuioiua**
Previously, a bit of manual work was required to set up Stripe. This change adds the init:stripe task, which:

1. Automatically creates the "Premium tier" product.
2. Automatically creates the default billing portal configuration used in the /dashboard/manage-subscription route.
3. Prints the "Premium tier" product price ID to be copied into constants.ts.

This dramatically reduces the time and steps needed and the chance of errors in setting up Stripe.

Contributors, please feel free to test and provide feedback.

### CODE REVIEW

**Review by lino-levan — COMMENTED** [review]
This looks fine as a v1, but I'm wondering if providing some flags would make sense? Not sure if it is in scope for this "getting started" script.

**Comment by iuioiua** [comment]
> This looks fine as a v1, but I'm wondering if providing some flags would make sense?

What do you suggest?

> Not sure if it is in scope for this "getting started" script.

It was deliberately written to replace instructions with automatic setup. Why would that be?

File: init/stripe.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import type { Stripe } from "stripe";
+import { SITE_DESCRIPTION } from "@/constants.ts";
+import "std/dotenv/load.ts";
+import { stripe } from "@/utils/stripe.ts";
+
+async function createPremiumTierProduct(stripe: Stripe) {
+  return await stripe.products.create({
+    name: "Premium tier",
```
Comment (lino-levan) [inline init/stripe.ts:9]: I'm wondering if the stuff in this object should be configurable? Does that make any sense?

**Comment by lino-levan** [comment]
That seems good enough for now. LGTM

File: init/stripe.ts (hunk 1 above)
Comment (iuioiua) [reply init/stripe.ts:13]: Good suggestion. Though, we'll leave this as-is for now in the spirit of prioritising ease of setup over customizability.

File: README.md
Comment (niklasmtj) [inline README.md:1]: What if we keep the manual instructions but put the automated script first in the chapter. So people who would like to do those steps manually have a guidance.

**Review by niklasmtj — APPROVED** [review]
Looks good! I think this will prevent a lot of headache for people starting out with Stripe

File: README.md [hunk 2]
```diff
@@ -71,68 +71,39 @@ This will automatically configure the database tables and their settings for us.
...
+   recommend using the test key for your development environment.
+2. Run `deno task init:stripe` and follow the instructions. This automatically
+   creates your "Premium tier" product and configures the Stripe customer
+   portal.
 
-3. Next, head over to
-   [your Stripe dashboard settings](https://dashboard.stripe.com/test/apikeys)
-   to copy the `STRIPE_SECRET_KEY` into your `.env` file. We recommend using the
-   test key for your development environment.
+> Note: go to [init/stripe.ts] if you'd like to learn more about how the
```
Comment (iuioiua) [inline README.md:82]: @niklasmtj, regarding https://github.com/denoland/saaskit/pull/93#discussion_r1168232989, I'd rather have one or the other. I've added this comment to point people in the right direction if they want to learn more. At some point, I'll include technical details as JSDoc comments in the implementation to give an even more precise idea of what's happening. Do you think this addresses your concern?

Comment (niklasmtj) [reply README.md:82]: Sounds good to me 👍🏼

**Comment by iuioiua** [comment]
I just tested this locally, and it works! 🥳

**Pull request merged**